### PR TITLE
[release/2*] Move to Azure DevOps

### PR DESCRIPTION
**Pull request by mmitche**
- Autogenerated off of Jenkins configuration
- Applies to release/2.1* and release/2.2* branches
- Removed ubuntu 14.04 configuration

### CODE REVIEW

File: .vsts-ci.yml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+pr:
+- release/2.1*
+- release/2.2*
+
+jobs:
+- job: debug_osx
+  displayName: OSX Debug
+  pool:
+    vmImage: macOS-10.13
+  steps:
```
Comment (chcosta) [inline .vsts-ci.yml]: Should add this step.  I noticed that repos that don't end up with dirty enlistments at times

```yaml
      - checkout: self
        clean: true
```

Comment (mmitche) [reply .vsts-ci.yml]: Thanks

**Review by chcosta — APPROVED** [review]
Seems reasonable to me.  Obviously could be cleaned up a bit (use a "matrix" strategy), but the repo owner could own making this prettier if they care.

File: .vsts-ci.yml (hunk 1 above)
Comment (mmitche) [reply .vsts-ci.yml]: I have Build.Repository.Clean as a global variable in the YAML (at the top).  I think this does the same thing. Any issue with that?

**Comment by mmitche** [comment]
> Seems reasonable to me. Obviously could be cleaned up a bit (use a "matrix" strategy), but the repo owner could own making this prettier if they care.

Yeah. For sure.  It's autogenerated and I don't expect it to change much over time.

File: .vsts-ci.yml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+- release/2.1*
+- release/2.2*
+
+jobs:
+- job: debug_osx
+  displayName: OSX Debug
+  pool:
+    vmImage: macOS-10.13
+  variables:
+    DOTNET_RUNTIME_ID: osx.10.12-x64
```
Comment (mmitche) [inline .vsts-ci.yml]: @peterhuene This is the correct RID?

Comment (peterhuene) [reply .vsts-ci.yml]: That's the RID I use to run tests locally on my MBP.

**Comment by mmitche** [comment]
@peterhuene Looks good?

**Review by peterhuene — APPROVED** [review]

**Pull request merged**